packages/tasks/src/hardware.ts

```diff
@@ -108,6 +108,10 @@ export const SKUS = {
 				tflops: 30.6,
 				memory: [12, 10],
 			},
+			"RTX Titan": {
+				tflops: 16.3,
```

Comment:
When based on FP16 performance (as mentioned in the comments further above), this should be the right value
(checked on https://www.techpowerup.com/gpu-specs/titan-rtx.c3311)
```suggestion
				tflops: 32.62,
```